src/typeDefs/dbSchema.ts

```diff
@@ -26,6 +26,12 @@ export type Submission = Omit<gqlType.Submission, 'facility' | 'healthcareProfes
     healthcareProfessionals: gqlType.HealthcareProfessionalSubmission[]
 }
 
+export type AuditLog = Omit<gqlType.AuditLog, 'id' | 'updatedDate'>
+
+export type AuditLogResult = {
+    isSuccesful: boolean
+}
```

Comment:
I made the decision to make the type in the dbSchema since I think we actually never will need to return this to the client. Since the creation of an audit log will happen when another action is triggered like creating healthcare professional